src/ucp/proto/proto_common.c

```diff
@@ -53,6 +55,16 @@ void ucp_proto_common_lane_priv_init(const ucp_proto_common_init_params_t *param
     } else {
         lane_priv->rkey_index = UCP_NULL_RESOURCE;
     }
+
+    /* Get max IOV from UCT capabilities */
+    iface_attr  = ucp_proto_common_get_iface_attr(&params->super, lane);
+    uct_max_iov = ucp_proto_common_get_iface_attr_field(iface_attr,
```

Comment:
I'm concerned about `ucp_proto_common_get_iface_attr_field`...

Reply:
yep, usually such macto accepts name, not offset. also this way it can work safely with different datatypes.

Reply:
ok, but since it will affect all existing protocols i prefer to do it in separate PR, to not add too many changed lines in this PR, EDYT?